### ISSUE TITLE
Ionic 4 Beta: iOS' status bar with header overlapped return

### ISSUE THREAD

**Issue by teerasej** [ionic-team/ionic#15075]
# Bug Report

**Ionic Info**]

```
Ionic:

   ionic (Ionic CLI)          : 4.0.3 (/usr/local/lib/node_modules/ionic)
   Ionic Framework            : @ionic/angular 4.0.0-beta.0
   @angular-devkit/core       : 0.7.0-rc.3
   @angular-devkit/schematics : 0.7.0-rc.3
   @angular/cli               : 6.0.8
   @ionic/ng-toolkit          : 1.0.0
   @ionic/schematics-angular  : 1.0.1

Cordova:

   cordova (Cordova CLI) : 8.0.0
   Cordova Platforms     : none

System:

   Android SDK Tools : 26.1.1
   ios-deploy        : 1.9.2
   ios-sim           : 6.1.2
   NodeJS            : v8.11.2 (/usr/local/bin/node)
   npm               : 6.3.0
   OS                : macOS High Sierra
   Xcode             : Xcode 9.4.1 Build version 9F2000
```

**Describe the Bug**
In Ionic 3, there is a space between iOS' status bar and the title of header component. it works both on ionic serve and real device.

But after try out with Ionic 4 beta, I found the header and status bar has overlapped. 

It happens both with `ionic serve` and on real device.

<a href="https://ibb.co/dQiEfe"><img src="https://preview.ibb.co/gETQmK/2192d3bb8f86fd94779cba0b5b716cb2a3413898.jpg" alt="2192d3bb8f86fd94779cba0b5b716cb2a3413898" border="0"></a>

**Steps to Reproduce**
1. create new project with `ionic start --type=angular`
2. In the project directory, run `ionic serve --lab`, found the overlapped.
3. Run `ionic cordova prepare ios`
4. Run iOS platform with Xcode, found the overlapped.

**Expected Behavior**
Header should get more space from status bar as it should.

**Additional Context**
This problem happened before in Ionic 2, but it should been fixed since Ionic 3.

**Comment by brandyscarney** [comment]
Thanks for the issue! Could you please try with latest `4.0.0-beta.1` and let me know if you still have the problem? This should be fixed in that release.

**Comment by teerasej** [comment]
@brandyscarney Thank you. I tried to create new project with `ionic start` and I found out this bug has been fixed with `4.0.0-beta.1`! 

Just one more question, how to update existing project (`4.0.0-beta.0`) to new version (`4.0.0-beta.1`)? 

I see in dependencies section of `package.json` still show `@ionic/angular": "^4.0.0-beta.0"`. So I wonder how to update it to new version.

**Comment by brandyscarney** [comment]
@teerasej You can run the following command from the root of your project to install `beta.2` (released today):

```
npm i @ionic/angular@4.0.0-beta.2
```

More info on `npm install`: https://docs.npmjs.com/cli/install

I'm going to close this issue since it is resolved, but thank you for reporting it!
